### PR TITLE
add fac11 week 6 scg

### PR DESCRIPTION
**Pull request by ConchobarMacNessa**
Added the stop go continue for fac11 week 6
relates #69

### CODE REVIEW

File: stop-go-continue/fac-11/cohort-week-6.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    + People find it super annoying.
+    + Aisha says it's hard when people need to partner up, ruins everyone's flow
+    + generally everyone agrees.
+    + Jen: of course people will be late, stuff happens, but if it's a habitual thing it's not cool
+    + Rachael: if you're half an hour late all the time, that adds up - people fall behind and it's up to everyone else to catch them up
+    + everyday lateness should not happens
+    + if we start lunch later because a workshop overruns, we should make it clear that we'll start later after lunch
+    + Max: it's fair to be frustrated by lateness, but it's genuinely harder for some people
+    + Rachael: this is a collaborative course, if someone has missed out on content, it creates a burden for the others. It's fine if you catch up on your own time, but it's not fair/easy on others in the group
+    + Jen S: why is it happening? Tom: it's started and now ingrained, plus the structure of the day is often broken with workshops overrunning
```
Comment (jsms90) [inline stop-go-continue/fac-11/cohort-week-6.md]: new line

File: stop-go-continue/fac-11/cohort-week-6.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    + Rebeca: we should tell people it starts at quarter to nine.
+    + Yahia: set your alarm twenty mins earlier. Being early is not a problem. Being late is.
+    + Yahia: if another workshop overruns we should make sure everyone knows when we'll start again.
+    + Alexis: We should set a punishment for late people. They have to buy milk/clean.
+    + Dak: in that case people who are late will just take that on as their job and still be late.
+    + Aisha: I'm always late for everything, but I purposely try to get here early.
+    + Alexis: there's no excuse for it. If this were a job and you were always late, you'd just lose get fired.
+    + Rebeca: lots of people in this cohort live really far away. Rachael: people have to commute further to work, you just have to value your own and other people's time.
```
Comment (jsms90) [inline stop-go-continue/fac-11/cohort-week-6.md]: new line

File: stop-go-continue/fac-11/cohort-week-6.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    + Jen: there's a potential change happening to the student document where you have to say in gitter if you're going to be late. Tell your team, it's important.
++ Being flexible about the time the workshops last
+    + Tom: It seems like at the start of the course the workshops fitted into their alloted times nicely, it seems like there's been some slippage, not just this week, the timing has become more flexible, which throws things off.
+    + Rebeca: most people had not even finished task 2/3 at the end. I liked that when we came back from lunch we had a chance to go back to it. (people did half an hour on the workshop during research). Aisha: it was really important to have that extra time, it was crucial to our project.
```
Comment (jsms90) [inline stop-go-continue/fac-11/cohort-week-6.md]: new line for Aisha's comment

File: stop-go-continue/fac-11/cohort-week-6.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+  + Constantina: should we have the five minute warning also be a place to discuss if we're going to overrrun?
++ Start research when we've discussed projects (for context)
+  + Rachael: I'm happy to discuss this another week when we have time. we get introduced to projects at the end of the day.
+  + Max: it could be an issue because once you get to your projects, you don't want to think about anything else. but that's the only negative.
+  + Aisha: Think this could be a good idea, all you know is the project idea, you don't know exactly what your project will be, so it's good
+  + Tom: we'd just do the research thinking about how it's applicable to our projects, not researching in general. Max disagrees with this, if we do it just for the projects it'll sink in more.
+  + Jen: maybe introduce the project during learning outcomes. Lots of agreement.
+  + Bart: I doubt it would solve the problem of research afternoons.
+  + Rachael: maybe we should do the research once we've started work on the projects. We could jigsaw and do a sprint during project time. People like this idea.
+  + Aisha: can we test this next week? Jen: I'll have to discuss that with future mentors.
```
Comment (jsms90) [inline stop-go-continue/fac-11/cohort-week-6.md]: new line

File: stop-go-continue/fac-11/cohort-week-6.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+  + Rachael: might be worth having an overview of the files we're going to write before hand then going over them after. It's a lot to take on at once if we type and learn.
+  + Max: with code alongs, less is more. There are always going to be issues. Maybe we should have a small code along, with an extension bit.  
+  + Yahia: we should have more mentors if there is a code along.
+  + Rebeca: we seem to be fast when we have structure, maybe follow a readme rather than have a code along
+  + others really like code alongs though
+  + Rachael: people do things at different speeds, so maybe we should have written inscructions/ a video you can go back to.
+  + Tom: Yvonne's code along went well, they may have used some kind of techniques to make that go well. Romanizer also went well. Should ask people who did good code alongs.
+  + Aseel: while Yvonne was doing code along, Zooey was troubleshooting, and Oli made a video. that's why it worked. Others really liked the video. Yahia: video takes off the pressure of typing along.
```
Comment (jsms90) [inline stop-go-continue/fac-11/cohort-week-6.md]: new line for Yahia's comment

File: stop-go-continue/fac-11/cohort-week-6.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+  + Rebeca: it's always good to have instructions to go back to. Always good to have mentors supporting the lead code alonger.
+  + Yahia: Fac could package and sell these videos as courses.
++ examples for db tests, and no guidance for mock database and travis
+  + Constantina: there was a lot of issues, it was hard to find resources and during projects there was a lot of confusion.
+  + Rebeca: I was on the testing research team, I didn't learn anything from it though.
+  + Aisha: Checking prompt questions for research topics. The testing db research topic asked for us to look for tools, and we spent too long looking into this, the only way we were able to get anywhere was in asking mentors to help. So we need to clarify this topic further.
+  + Tom: maybe we should produce a resource on how to test dbs.
+  + Rachael: look at the kidney bean project to learn how to test dbs.
+  + Constantina: resources for this topic were too hard to find. With server testing we had a workshop and written examples, this week we didn't.
+  + Bart: should we write down the steps that we went through, could we get input from more experienced people? Jen: put it on fac github.
```
Comment (jsms90) [inline stop-go-continue/fac-11/cohort-week-6.md]: new line

File: stop-go-continue/fac-11/cohort-week-6.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
++ consolidation of sql language workshop
+  + Max: at the end of the first day, it would have been nice to have had a quick recap of the main sql queries. Like a cheatsheet.
++ non boozy social things
+  + everyone agrees
++ being engaged with whatever you're delivering enthusiastically
+  + Rebeca: the message is always better if the message is delivered with enthusiasm. EG when we had some code alongs or workshops, i find I pay more attention if the person that is delivering the workshop is enjoying the task.
+  + Tom: I agree. if someone's having a bad day, or is a person who is not naturally like that, what do we do? Rebeca: we're all here to improve, not just coding.
```
Comment (jsms90) [inline stop-go-continue/fac-11/cohort-week-6.md]: new line

File: stop-go-continue/fac-11/cohort-week-6.md [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+  + Aseel: sometimes if someone has a quiet demeaner, it's going to be different, but there is a level of disinterest coming across. We can sense how it's been delivered.
+  + Tom: you're talking about an intangeable quality. How does someone go about changing that?
+  + Rachael: how much training are we giving mentors? Are we nurturing presentation skills? Alexis: Fridays are an opportunity to improve. Rachael: more direct tips need to be given.
```
Comment (jsms90) [inline stop-go-continue/fac-11/cohort-week-6.md]: 2 new lines

File: stop-go-continue/fac-11/cohort-week-6.md [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+    + Jen S: why is it happening? Tom: it's started and now ingrained, plus the structure of the day is often broken with workshops overrunning
+    + Rebeca: we should tell people it starts at quarter to nine.
+    + Yahia: set your alarm twenty mins earlier. Being early is not a problem. Being late is.
+    + Yahia: if another workshop overruns we should make sure everyone knows when we'll start again.
+    + Alexis: We should set a punishment for late people. They have to buy milk/clean.
+    + Dak: in that case people who are late will just take that on as their job and still be late.
+    + Aisha: I'm always late for everything, but I purposely try to get here early.
+    + Alexis: there's no excuse for it. If this were a job and you were always late, you'd just lose get fired.
+    + Rebeca: lots of people in this cohort live really far away. Rachael: people have to commute further to work, you just have to value your own and other people's time.
+    + Jen: there's a potential change happening to the student document where you have to say in gitter if you're going to be late. Tell your team, it's important.
```
Comment (jsms90) [inline stop-go-continue/fac-11/cohort-week-6.md]: Jen S*

File: stop-go-continue/fac-11/cohort-week-6.md [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+  + Jen: if an individual struggles they can set a timer, it's not the mentors' responsibilities to tell you. Maybe a mentor should say 'this is finishing at x time'. Tom says it's not about not knowing when we're finishing, more just we need a five minute reminder.
+  + Constantina: should we have the five minute warning also be a place to discuss if we're going to overrrun?
++ Start research when we've discussed projects (for context)
+  + Rachael: I'm happy to discuss this another week when we have time. we get introduced to projects at the end of the day.
+  + Max: it could be an issue because once you get to your projects, you don't want to think about anything else. but that's the only negative.
+  + Aisha: Think this could be a good idea, all you know is the project idea, you don't know exactly what your project will be, so it's good
+  + Tom: we'd just do the research thinking about how it's applicable to our projects, not researching in general. Max disagrees with this, if we do it just for the projects it'll sink in more.
+  + Jen: maybe introduce the project during learning outcomes. Lots of agreement.
```
Comment (jsms90) [inline stop-go-continue/fac-11/cohort-week-6.md]: Jen S*

File: stop-go-continue/fac-11/cohort-week-6.md (hunk 4 above)
Comment (jsms90) [inline stop-go-continue/fac-11/cohort-week-6.md]: Jen S*

File: stop-go-continue/fac-11/cohort-week-6.md (hunk 6 above)
Comment (jsms90) [inline stop-go-continue/fac-11/cohort-week-6.md]: Jen S*

File: stop-go-continue/fac-11/cohort-week-6.md [hunk 11]
```diff
@@ -0,0 +1,94 @@
...
+  + Tom: I agree. if someone's having a bad day, or is a person who is not naturally like that, what do we do? Rebeca: we're all here to improve, not just coding.
+  + Aseel: sometimes if someone has a quiet demeaner, it's going to be different, but there is a level of disinterest coming across. We can sense how it's been delivered.
+  + Tom: you're talking about an intangeable quality. How does someone go about changing that?
+  + Rachael: how much training are we giving mentors? Are we nurturing presentation skills? Alexis: Fridays are an opportunity to improve. Rachael: more direct tips need to be given.
+  + Rebeca: what if when we're mentors, what if we meet at the weekend and present our topics to each other and give direct feedback?
+  + Aisha: what if after presentations we get back into our groups and discuss how our presentation went and how to improve? lots of agreement.
+  + Jen: we used to have team scg after presentations, specifically for this reason.
```
Comment (jsms90) [inline stop-go-continue/fac-11/cohort-week-6.md]: Jen S*

**Comment by ConchobarMacNessa** [comment]
@jsms90 Just put these changes in now

**Pull request merged**